support-models/src/main/scala/com/gu/support/workers/GiftRecipient.scala

```diff
@@ -39,10 +42,8 @@ object GiftRecipient {
   private val discriminatorWeekly = "Weekly"
   private val discriminatorDigitalSubscription = "DigitalSubscription"
 
-  implicit val e1 = circeDiscriminator.add(deriveEncoder[WeeklyGiftRecipient], discriminatorWeekly)
-  implicit val d1 = deriveDecoder[WeeklyGiftRecipient]
-  implicit val e2 = circeDiscriminator.add(deriveEncoder[DigitalSubscriptionGiftRecipient], discriminatorDigitalSubscription)
-  implicit val d2 = deriveDecoder[DigitalSubscriptionGiftRecipient]
```

Comment:
small improvement to `CirceDiscriminator` to save a line of code for each subtype